generators/server/templates/src/main/java/package/config/WebConfigurer.java.ejs

```diff
@@ -145,7 +145,7 @@ public class WebConfigurer implements <% if (!reactive) { %>ServletContextInitia
             // IE issue, see https://github.com/jhipster/generator-jhipster/pull/711
             mappings.add("html", MediaType.TEXT_HTML_VALUE + ";charset=" + StandardCharsets.UTF_8.name().toLowerCase());
             // CloudFoundry issue, see https://github.com/cloudfoundry/gorouter/issues/64
```

Comment:
This comment can be removed since it no longer applies

Reply:
Yes apologies. Should I create a second commit or amend the previous one?

Reply:
My first contribution. Starting small was a good idea. haha

Reply:
@AlexandreCassagne : just amend the previous one ;)
And congrats for your first contribution !

Reply:
@ruddell I amended my previous pull request.

Reply:
@murdos  thank you ;-)